packages/tests/components/SfTextarea/SfTextarea.cy.tsx

```diff
@@ -0,0 +1,108 @@
+/// <reference path="../../../../../node_modules/@percy/cypress/types/index.d.ts" />
```

Comment:
after Ania [PR](https://github.com/vuestorefront/storefront-ui/pull/2891) this line can be removed, please rebase or remove it :)

Reply:
removed